zipline/pipeline/loaders/blaze/core.py

```diff
@@ -206,8 +206,13 @@
 get__name__ = op.attrgetter('__name__')
 
 
-class ExprData(namedtuple('ExprData', 'expr deltas checkpoints odo_kwargs')):
-    """A pair of expressions and data resources. The expresions will be
+class ExprData(
+    namedtuple(
+        'ExprData',
+        'expr deltas checkpoints odo_kwargs apply_deltas_adjustments'
+    )
+):
```

Comment:
this makes it look like we are closing the class scope. define the super class above as `_expr_data_base`